src/js/common/schemaform/fields/FileField.jsx

```diff
@@ -130,7 +142,8 @@ export default class FileField extends React.Component {
                       </button>
                     </div>
                   }
-                  {!file.uploading && <span>{file.name}</span>}
+                  {!file.uploading && <p>{uiOptions.itemDescription}</p>}
+                  {!file.uploading && <span><strong>{file.name}</strong></span>}
```

Comment:
@goldenmeanie should the file name be bold in general now, or only for this form?